test/array_serializer_test.rb

```diff
@@ -39,15 +45,52 @@ def test_meta_and_meta_key_attr_readers
         assert_equal @serializer.meta_key, "the meta key"
       end
 
-      def test_json_key_when_resource_is_empty
```

Comment:
Isn't that a duplicated by https://github.com/rails-api/active_model_serializers/pull/1013/files#diff-4511bb323b8555a8103c7964418e0e40R69? I favored that version because it doesn't modify the Array class itself.